test/libsolidity/SolidityOptimizer.cpp

```diff
@@ -334,7 +334,7 @@ BOOST_AUTO_TEST_CASE(store_tags_as_unions)
 			numSHA3s++;
 	});
 // TEST DISABLED UNTIL 93693404 IS IMPLEMENTED
```

Comment:
Just scrolled through - It's unrelated to this PR, but I'm really wondering now what `93693404` means

Reply:
This comment is ancient history (written still in cpp-ethereum and only imported to this repo in #5) so it's probably an ID from whatever issue tracking system the project was using back then. I think it was Pivotal Tracker. There are traces of it in some old issues, e.g. https://github.com/ethereum/solidity/issues/125#issuecomment-146481290